graphql_objects.py

```diff
@@ -1,7 +1,17 @@
+from datetime import date
+from typing import Optional
+
+
 class DatesCheck:
 
     def __init__(
             self,
-            listing_id: int
+            listing_id: int,
+            is_available: bool,
+            checkin_date: Optional[date] = None,
+            checkout_date: Optional[date]= None,
     ):
         self.listing_id = listing_id
+        self.is_available = is_available
+        self.checkin_date = checkin_date
+        self.checkout_date = checkout_date
```

Comment:
Does it mean that your checkin and out dates in the final DatesCheck output can be optional? Does this make sense?